src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -29,10 +29,13 @@ export enum DeclarativeDataType {
 @Component({
 	selector: 'modelview-declarativeTable',
 	template: `
-	<table role=grid #container *ngIf="columns" class="declarative-table" [style.height]="getHeight()" [attr.aria-label]="ariaLabel">
+	<table role=grid #container *ngIf="columns" class="declarative-table" [attr.aria-label]="ariaLabel">
```

Comment:
Don't just remove setting the height style - we still want to allow people to control the height on this component if they wish.

You can just remove the specified height in the modify columns page instead : https://github.com/microsoft/azuredatastudio/blob/master/extensions/import/src/wizard/pages/modifyColumnsPage.ts#L140